app/models/user.rb

```diff
@@ -22,6 +24,24 @@ def digest string
       end
       BCrypt::Password.create(string, cost: cost)
     end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    @remember_token = User.new_token
+    User.update self.id, remember_digest: User.digest(remember_token)
```

Comment:
ngoài update ra thì không còn cách nào mà nó validate trước khi update nữa anh ạ
http://www.davidverhasselt.com/set-attributes-in-activerecord/